src/scicookie/profiles/base.yaml

```diff
@@ -87,15 +87,15 @@ build_system:
   default: poetry
   # first choice is the default for the UI
   choices:
-    - poetry
-    - flit
-    - mesonpy
-    - setuptools
-    - pdm
-    - hatch
-    - maturin
-    - scikit-build-core
-    - pybind11
+  - poetry
+  - flit
+  - mesonpy
+  - setuptools
+  - pdm
+  - hatch
+  - maturin
+  - scikit-build-core
+  - pybind11
```

Comment:
Ditto

Reply:
Implemented

Reply:
Not resolved, marking this as unresolved, see the indentation